DOCS/openfga-how-to.md

```diff
@@ -0,0 +1,196 @@
+HTTP/1.1 201 Created
+Content-Length: 143
+Content-Type: application/json
+Date: Mon, 08 Apr 2024 14:06:26 GMT
+Vary: Origin
+X-Http-Code: 201
+X-Request-Id: 1da6c68d-d3fe-4e01-b957-19da07cb5270
+
+{
+    "created_at": "2024-04-08T14:06:26.848060Z",
+    "id": "01HTZ0G7GZ4QEKHV82TV59H6ES",
+    "name": "openfga-demo",
+    "updated_at": "2024-04-08T14:06:26.848060Z"
+}
+```
+
+
+### Observability integration
+
+This OpenFGA operator integrates with [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack) (COS) bundle.
+It comes with a Grafana dashboard as well as Loki and Prometheus alert rules for basic common scenarios.
+To integrate with the COS bundle, after you [deploy it](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s#heading--deploy-the-cos-lite-bundle), you can run:
+
+
+```shell
+$ juju integrate openfga:grafana-dashboard grafana:grafana-dashboard
+$ juju integrate openfga:metrics-endpoint prometheus:metrics-endpoint
+$ juju integrate loki:logging openfga:log-proxy
+```
+
+
+## Scaling up/down
+
+To scale the OpenFGA server we can exploit `juju scale-application`
+
+
+```shell
+$ juju scale-application openfga-k8s 5
+openfga-k8s scaled to 5 units
+```
+
+In due time, we should be able to see that all the requested units have come up successfully
+
+
+```shell
+$ juju status --relations                               
+Model    Controller          Cloud/Region        Version  SLA          Timestamp
+openfga  microk8s-localhost  microk8s/localhost  3.1.7    unsupported  16:37:36+02:00
+
+App             Version  Status  Scale  Charm           Channel  Rev  Address         Exposed  Message
+openfga-k8s              active      5  openfga-k8s                0  10.152.183.172  no       
+postgresql-k8s  14.11    active      1  postgresql-k8s  14/edge  233  10.152.183.177  no       Primary
+
+Unit               Workload  Agent  Address       Ports  Message
+openfga-k8s/0*     active    idle   10.1.245.154         
+openfga-k8s/1      active    idle   10.1.245.141         
+openfga-k8s/2      active    idle   10.1.245.144         
+openfga-k8s/3      active    idle   10.1.245.155         
+openfga-k8s/4      active    idle   10.1.245.131         
+postgresql-k8s/0*  active    idle   10.1.245.156         Primary
+
+Integration provider           Requirer                       Interface          Type     Message
+openfga-k8s:peer               openfga-k8s:peer               openfga-peer       peer     
+postgresql-k8s:database        openfga-k8s:database           postgresql_client  regular  
+postgresql-k8s:database-peers  postgresql-k8s:database-peers  postgresql_peers   peer     
+postgresql-k8s:restart         postgresql-k8s:restart         rolling_op         peer     
+postgresql-k8s:upgrade         postgresql-k8s:upgrade         upgrade            peer    
+```
+---
+
+
+## Extra integrations
+
+
+### New `openfga` interface:
```

Comment:
I don't like this section. I would suggest to either rename it to
```suggestion
##  `Openfga` interface
```

Or to remove it altogether. The things documented in here are common to every charm library and not specific to openfga